cmd/stack_package.go

```diff
@@ -673,3 +682,17 @@ func ApplyTemplating(stackPath string, templateMetadata interface{}) error {
 	return nil
 
 }
+
+func removePackagingFolder(stackPath string) error {
```

Comment:
This function looks good, thanks! Just a suggestion would be to rename it to `removePackagingFolderIfExists` or even rename it to `removeFolderIfExists` and move it to `utils.go`, but I will leave this to your judgement!

Reply:
I agree I like this more generic approach.  I will do that now.